CoreScriptsRoot/CoreScripts/PurchasePromptScript3.lua

```diff
@@ -1590,7 +1590,7 @@ end
 function onInputChanged(inputObject)
 	local input = inputObject.UserInputType
 	local inputs = Enum.UserInputType
-	if valueInTable(input, {inputs.Gamepad1, inputs.Gamepad2, inputs.Gamepad3, inputs.Gamepad4}) then
+	if not IsVRMode and valueInTable(input, {inputs.Gamepad1, inputs.Gamepad2, inputs.Gamepad3, inputs.Gamepad4}) then
```

Comment:
In VR we have gaze select, so if you put a B hint on the Cancel button but pressing A activates it... (and also the converse, pressing A would activate the Buy button even though you wanted to cancel.)